src/newLaunch/baseStage.scss

```diff
@@ -140,15 +140,6 @@
   .importantText {
```

Comment:
(just general comment)
@giladt 
Can you cross-check with PR #773 
there were some unrelated changes, but they seem to be good to keep